pkg/controller/controller_broker.go

```diff
@@ -314,7 +314,7 @@ func (c *controller) reconcileClusterServiceBroker(broker *v1alpha1.ClusterServi
 		}
 
 		glog.V(4).Infof("ClusterServiceBroker %q: converting catalog response into service-catalog API", broker.Name)
-		serviceClasses, servicePlans, err := convertCatalog(brokerCatalog)
+		payloadServiceClasses, payloadServicePlans, err := convertCatalog(brokerCatalog)
```

Comment:
I think payload name makes it harder to follow the function later on. Might be better to call these something that clearly says these are BROKER classes and the maps below are STORAGE, or SC_VIEW classes/maps, or something, because current / payload makes it trickier to keep the context right for me.

Reply:
I guess 'existing' below is fine if payload is something different, but maybe I'm just being wonky about this. new / old? :) Don't feel strongly, but found myself scrolling back and forth looking at which classes are these again.

Reply:
I'm totes happen to change the names